packages/adblocker/src/filters/network.ts

```diff
@@ -1369,11 +1381,19 @@ export default class NetworkFilter implements IFilter {
   }
 
   public getRedirectResource(): string {
-    return this.getRedirect().split(':')[0];
+    const priorityIndex = this.getRedirect().lastIndexOf(':');
+    if (priorityIndex === -1) {
+      return this.getRedirect();
+    }
+    return this.getRedirect().slice(0, priorityIndex);
```

Comment:
Nit. Let's avoid calling the same method 3 times (although it's very fast)
```suggestion
    cons redirect = this.getRedirect();
    const priorityIndex = redirect.lastIndexOf(':');
    if (priorityIndex === -1) {
      return redirect;
    }
    return redirect.slice(0, priorityIndex);
```